Assets/UI/worldinput_CQUI.lua

```diff
@@ -0,0 +1,559 @@
+include("WorldInput");
+
+-- ===========================================================================
+-- Cached Base Functions
+-- ===========================================================================
+BASE_CQUI_UpdateDragMap            = UpdateDragMap;
+BASE_CQUI_RealizeMovementPath      = RealizeMovementPath;
+BASE_CQUI_OnUnitSelectionChanged   = OnUnitSelectionChanged;
+BASE_CQUI_OnInputActionTriggered   = OnInputActionTriggered;
+BASE_CQUI_DefaultKeyDownHandler    = DefaultKeyDownHandler;
+BASE_CQUI_DefaultKeyUpHandler      = DefaultKeyUpHandler;
+BASE_CQUI_OnDefaultKeyDown         = OnDefaultKeyDown;
+BASE_CQUI_OnDefaultKeyUp           = OnDefaultKeyUp;
+BASE_CQUI_OnPlacementKeyUp         = OnPlacementKeyUp;
+BASE_CQUI_ClearAllCachedInputState = ClearAllCachedInputState;
+BASE_CQUI_LateInitialize           = LateInitialize;
+BASE_CQUI_Initialize               = Initialize;
+
+-- ===========================================================================
+-- CQUI Members
+-- ===========================================================================
+local CQUI_HOTKEYMODE_ENHANCED    = 2;
+local CQUI_HOTKEYMODE_CLASSIC     = 1;
+local CQUI_HOTKEYMODE_STANDARD    = 0;
+
+local m_isInputBlocked  :boolean = false;
+local CQUI_cityview     :boolean = false;
+local CQUI_hotkeyMode   :number  = CQUI_HOTKEYMODE_ENHANCED;
+local CQUI_isShiftDown  :boolean = false;
+local CQUI_isAltDown    :boolean = false;
+local CQUI_isCtrlDown   :boolean = false;
+
+function CQUI_OnSettingsUpdate()
+  CQUI_hotkeyMode = GameConfiguration.GetValue("CQUI_BindingsMode");
+end
+
+LuaEvents.CQUI_SettingsUpdate.Add( CQUI_OnSettingsUpdate );
+LuaEvents.CQUI_SettingsInitialized.Add( CQUI_OnSettingsUpdate );
+
+-- ===========================================================================
+--  VARIABLES
+-- ===========================================================================
+local CQUI_ShowDebugPrint = false;
+
+-- ===========================================================================
+--CQUI setting control support functions
+-- ===========================================================================
+function print_debug(str)
+  if CQUI_ShowDebugPrint then
+    print(str);
+  end
+end
+
+-- ===========================================================================
+-- CQUI Base Extension Functions
+-- Each of these functions call the Base Function found in WorldInput.lua
+-- ===========================================================================
+
+-- ===========================================================================
+function UpdateDragMap()
+  if g_isMouseDragging then
+    -- Event sets a global in PlotInfo so tiles are not purchased while dragging
+    LuaEvents.CQUI_StartDragMap();
+  end
+
+  return BASE_CQUI_UpdateDragMap();
+end
+
+-- ===========================================================================
+function RealizeMovementPath(showQueuedPath:boolean)
```

Comment:
Looks like the ability to see unit path on hover over unit is being removed. Is it intentional? If yes need to remove CQUI option that says hover on unit will show unitpath and also clear up related function calls (CQUI_OnUnitFlagPointerEntered in unitflagmanager_CQUI.lua).
My preference is to keep the hover functionality (unless the unmodded game has that functionality now), it comes in handy when planning movements with lots of units.

Reply:
yeah, I was wondering about that, and had intended to look into it.  Good catch.  I would also prefer the hover show the planned movement.

Reply:
Actually - a little confused here... are you saying the base game is removing the show path on hover?

Reply:
The base game doesn't have the ability to show path on hover. It was added by Azurency by adding the function I mentioned in original comment and by adding a new parameter for this function. Your modification looks like it removes the CQUI ability to show unit path without selecting the unit.

Reply:
Aaah - okay, it wasn't anything specific in the RealizeMovementPath function... I think it may have been that I was calling the base initialize after Adding methods for the LuaEvents.  Checking into that now.

Reply:
Argh, ok, I see it.  Was hoping to avoid having a huge function in there, but it seems in order to achieve this, we need to have the huge function in there.

Reply:
had to put in the entire function, but it works now!